portfolio/src/main/webapp/style.css

```diff
@@ -3,7 +3,93 @@
   margin-right: auto;
   width: 650px;
 }
+/* Images Style */
+.image-thumbnails {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.image-thumbnails img {
+  margin: 10px;
+  max-height: 200px;
+  max-width: 200px;
+}
+
+.image-thumbnails img:hover {
+  border: thick solid black;
+}
+
+#random-image-container img {
+  margin-top: 20px;
+}
+
 
 #greeting-container {
   margin-top: 20px;
 }
+/* Menu Style */
+ body{
+    margin: 0px;
+}
+
+.menu-main{
```

Comment:
Please be consistent about how you space and format files (including CSS). In particular, you sometimes have a space before the `{` and sometimes don't. I'd prefer always having one.

Ideally, there would be automatic formatting tools for all the programming languages that one uses. [Stylelint](https://stylelint.io) looks reasonable for CSS.

In terms of a formatting convention: Google has prepared an [HTML/CSS style guide](https://google.github.io/styleguide/htmlcssguide.html) that's a nice convention.